config/zones/MY.yaml

```diff
@@ -0,0 +1,11 @@
+country: MY
+disclaimer: 'This zone is partially using fully estimated data from MY-EM'
+estimation_method:
+  CONSTRUCT_BREAKDOWN
```

Comment:
Do you think it would make sense to make a follow up PR and change all of these for country level to `estimation_method: SPATIAL_AGGREGATE`?

Reply:
> Do you think it would make sense to make a follow up PR and change all of these for country level to `estimation_method: SPATIAL_AGGREGATE`?

Either that or we just check that it has subzones, but maybe that's not strict enough?
If you want to change them now that is fine with me though. Then we can look into optimizing it later.